styles/modules/modals/_settings.scss

```diff
@@ -96,4 +96,12 @@
       }
     }
   }
+
+  .previewBox {
+    min-width: 1em;
+    min-height: 1em;
+    width: auto;
+    height: 60%;
+    margin: 20%;
+  }
```

Comment:
Since input elements are set to have 40px line heights, and this class won't be used in any other situation, it would be best to just set this element to have the same for height and width, like this:

```
height: $formElementHeight;
width: $formElementHeight;
```

If you also remove the margin, I think it looks quite nice with the above size (see screenshot). 

You can also set the border-width and border style here, as this will always have a border, rather than add a border class to the element.

![image](https://cloud.githubusercontent.com/assets/1584275/22796297/efeeb928-eec7-11e6-8917-dd096d22b92a.png)